.github/workflows/main.yml

```diff
@@ -97,5 +97,5 @@ jobs:
       - name: "Install dependencies"
         run: "composer update --no-suggest --no-interaction --prefer-dist --no-progress"
 
-      - name: "Composer Normalize"
-        run: "composer normalize"
+      - name: "Check code styles"
```

Comment:
I would keep the Composer normalize there. Its general style lint for `composer.json`. see: https://github.com/ergebnis/composer-normalize

Reply:
Ok, I will add both checks here.